docs/zh/maintain/monitoring.md

```diff
@@ -111,7 +138,7 @@ OpenMLDB exporter 是以 Python 实现的 prometheus exporter，核心是通过
 
 如何安装部署 prometheus, grafana 详见官方文档 [promtheus get started](https://prometheus.io/docs/prometheus/latest/getting_started/) 和 [grafana get started](https://grafana.com/docs/grafana/latest/getting-started/getting-started-prometheus/) 。我们建议使用 Docker 容器快速部署, 并且 Grafana >= 8.3, Prometheus >= 1.0.0 。
 
-OpenMLDB 提供了 prometheus 和 grafana 配置文件以作参考，详见 [OpenMLDB mixin](https://github.com/4paradigm/OpenMLDB/tree/main/monitoring/openmldb_mixin/README.md):
+OpenMLDB 提供了 prometheus 和 grafana 配置文件以作参考，详见 [OpenMLDB mixin](https://github.com/4paradigm/openmldb-exporter/tree/main/openmldb_mixin):
```

Comment:
Prometheus
Grafana

Reply:
but the monitoring code is still there https://github.com/4paradigm/OpenMLDB/tree/main/monitoring
shall we remove those code?

Reply:
> but the monitoring code is still there https://github.com/4paradigm/OpenMLDB/tree/main/monitoring shall we remove those code?

never mind, i saw it in another PR :-)